README.md

```diff
@@ -32,36 +104,42 @@ curl --request POST \
   --data 'token=__SLACK_TOKEN__&channel_name= __SLACK_CHANNEL__&text=somelane%20someargs'
 ```
 
-## ⚙️ Environment Variables
-
- To set the aforementioned environment variables with the real values on Heroku:
-
- * Go to your Heroku dashboard
- * Navigate to Settings
- * set the environment variables like `SLACK_TOKEN` etc
-
-## 🕹 Create a new Slack command
+## 🕹 Adding a new Slack command to the app
 
 If you need to create a new Slack command:
 
  1. Go to the Slack Commands config page for your team's Slack app: `https://api.slack.com/apps/<YourSlackAppID>/slash-commands`
  2. Click on "Create New Command"
    * Fill in the slash command (e.g. `/foo`)
-   * Enter `https://<appname>.herokuapp.com/<command>` as the request URL, replacing `<appname>` with the name of your Heroku app instance (e.g. `stevenson-bot`) and `<command>` by the command name
+   * Enter `https://<appname>.herokuapp.com/<command>` as the request URL, replacing `<appname>` with the name of your Heroku app instance (e.g. `stevenson-bot`) and `<command>` by the command name (e.g. `foo`)
    * Fill in the short description and the hint for the command
    * Hit "Save"
- 3. Open the project in Xcode and add a new the handler for the Slack command:
+ 3. Open the project in Xcode and add a new handler for the Slack command:
    * Open `commands.swift`
-   * Add a new `static let <commandName> = SlackCommand(...)` for your new command, using `<command>` as it's name and the `SLACK_TOKEN` environment variable
+   * Implement your command as a static function in the `SlackCommand` namespace:
+   
+   ```swift
+extension SlackCommand {
+   static let <command> = { 
+        SlackCommand(
+            name: "<command>", 
+            help: "...",
+            allowedChannels: [...],
+            run: { ... }
+        ) 
+    }
```

Comment:
This example seems like a mix of a static property and a static func 😅 

Since the rest of the doc below assumes it was declared as a function (`.<command>()`), better be consistent here then. (I've added 2 separate "GitHub suggestions" above since we can't make a GitHub suggestion on multiple lines at once, but those would need to be applied together to make sense)

Reply:
We currently have these commands defined as static closures, so I'd prefer to keep this example close to how it is in the implementation